lib/puppet/transaction/report.rb

```diff
@@ -326,7 +322,7 @@ def to_data_hash
     }
 
     # The following is include only when set
-    hash['master_used'] = hash['server_used'] = @server_used unless @server_used.nil?
+    hash['server_used'] = @server_used unless @server_used.nil?
```

Comment:
Sorry I missed this, I think we should always support reading the old value (and setting `server_used`).